openmc/data/neutron.py

```diff
@@ -326,6 +326,17 @@ def from_hdf5(cls, group_or_filename):
                     tgroup = group['total_nu']
                     rx.derived_products.append(Product.from_hdf5(tgroup))
 
+        # Build summed reactions.  Start from the highest MT number because high
+        # MTs never depend on lower MTs.
+        for mt_sum in sorted(SUM_RULES.keys())[::-1]:
```

Comment:
And actually, sorting on a dictionary uses key by default so you should be able to just do
```Python
sorted(SUM_RULES, reverse=True)
```